_userguides/Box_Panel.md

```diff
@@ -142,20 +142,20 @@ To view a list of Subscribers for a selected ticket, click on the **Subscribers*
 
 ![ ]()
 
-To have a written conversation with one of our Blue Box support team experts about the topic of this ticket, you can click the green **Post Reply** button, or you can use the live **Chat** feature by clicking on **Chat**. All live chat sessions are recorded and added to the ticket as a transcript.
+To have a written conversation with one of our Bluemix on OpenStack support team experts about the topic of this ticket, you can click the green **Post Reply** button, or you can use the live **Chat** feature by clicking on **Chat**. All live chat sessions are recorded and added to the ticket as a transcript.
 
-**Using the Chat Feature to chat with Blue Box Support**
+**Using the Chat Feature to chat with Bluemix on OpenStack Support**
 
 1. Click the **Chat** tab from your Dashboard. This action automatically directs you to a chat room with support staff. 
 2. From here, you are able to view the support staff members join your conversation on the left-hand side of the chat room.
 3. Once a support member joins, you can compose and send messages from the chat bar. If you need to enter a new line during your chat, press **Control/Command + Enter**.
 4. If you need to escalate your support issue, please call 1-800-613-4305 ext 1.
 
-**Note:** Each customer chat room is private. This means that only the contacts within your customer account and the Blue Box support staff have access to your transcript. No sensitive data (such as credit card numbers, or private keys) persists.
+**Note:** Each customer chat room is private. This means that only the contacts within your customer account and the Bluemix on OpenStack support staff have access to your transcript. No sensitive data (such as credit card numbers, or private keys) persists.
 
 ### Managing Billing 
 
 To view your contracts and monthly billing reports, click the **Account** tab in the Box Panel Dashboard, then click the **Billing** tab.
 
-For further billing support, please contact the Blue Box Support Team, and we will be happy to assist you.
-To add additional line items, such as expansion nodes, to your infrastructure, please contact the Blue Box Support Team. We will create a new monthly contract and help to streamline your setup process in any way that we can.
+For further billing support, please contact the Bluemix on OpenStack Support Team, and we will be happy to assist you.
+To add additional line items, such as expansion nodes, to your infrastructure, please contact the Bluemix on OpenStack Support Team. We will create a new monthly contract and help to streamline your setup process in any way that we can.
```

Comment:
This information actually is no longer accurate. Please delete the last 2 sentences, because we can no longer set up a new contract for our customers nor add line items.